src/flotilla/scheduler/doctor.py

```diff
@@ -3,12 +3,10 @@
 
 logger = logging.getLogger('flotilla')
 
-SERVICE_EXPIRY = 10
-
 
 class ServiceDoctor(object):
     def __init__(self, db, elb):
-        self._db = db
+        self.db = db
```

Comment:
I gets referenced twice in https://github.com/pebble/flotilla/blob/revision-json/src/flotilla/cli/revision.py#L262 too (for tests I'd just leave it private, lint-er be damned).

Not very pretty (I could have just as easily built a `db_cache` alongside `doctor_cache`), but this was close to deadline and my demo was the deployment pipeline; will tidy this up.